nixos/tests/installed-tests/xdg-desktop-portal.nix

```diff
@@ -9,9 +9,20 @@ makeInstalledTest {
   # Failed connect to PipeWire: Couldn't connect to PipeWire
   testConfig = {
     environment.variables = {
-      TEST_IN_CI = 1;
+      XDP_TEST_IN_CI = 1;
       XDG_DATA_DIRS = "${pkgs.xdg-desktop-portal.installedTests}/share/installed-tests/xdg-desktop-portal/share";
```

Comment:
This is probably redundant since https://github.com/flatpak/xdg-desktop-portal/commit/10523060f6c71018a7a253fc06f300201811f971